src/map/core/raw.rs

```diff
@@ -172,7 +172,7 @@ impl<'a, K, V> OccupiedEntry<'a, K, V> {
         // SAFETY: This is safe because it can only happen once (self is consumed)
         // and map.indices have not been modified since entry construction
         let index = unsafe { self.map.indices.remove(self.raw_bucket) };
```

Comment:
I would rather see this instead of `index.0`:
```suggestion
        let (index, _slot) = unsafe { self.map.indices.remove(self.raw_bucket) };
```

Reply:
Done.